shipper.py

```diff
@@ -55,11 +55,16 @@ def on_message(client, userdata, msg):
     data = json.loads(str(msg.payload));
     device = data['dev_eui']
     try:
+        # Try if we have base64-encoded JSON payload
         payload = json.loads(base64.b64decode(data['payload']))
         data.update(payload)
-    except:
+    except ValueError:
         payload = base64.b64decode(data['payload'])
-        data['raw'] = payload
+        try:
+            json.dumps(payload) # Try if we have base64 encoded payload
```

Comment:
This is needed in case the payload is just plain JSON - in this case the old code would die, because it would decode base64 and produce character sequences that can't easily be transformed to JSON any more, e.g. it would die with:
```console
  File "/usr/lib/python2.7/threading.py", line 801, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 2287, in _thread_main
    self.loop_forever()
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 1261, in loop_forever
    rc = self.loop(timeout, max_packets)
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 811, in loop
    rc = self.loop_read(max_packets)
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 1073, in loop_read
    rc = self._packet_read()
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 1475, in _packet_read
    rc = self._packet_handle()
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 1943, in _packet_handle
    return self._handle_publish()
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 2118, in _handle_publish
    self._handle_on_message(message)
  File "/env/lib/python2.7/site-packages/paho/mqtt/client.py", line 2274, in _handle_on_message
    self.on_message(self, self._userdata, message)
  File "/app/shipper.py", line 63, in on_message
    json.dumps(payload) # Try if we have base64 encoded payload
  File "/usr/lib/python2.7/json/__init__.py", line 244, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 201, in encode
    return encode_basestring_ascii(o)
UnicodeDecodeError: 'utf8' codec can't decode byte 0x96 in position 0: invalid start byte
```